python/examples/embedding/wasi-py-rs/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "wasi-py-rs"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+pyo3 = { version = "0.19.0", features = ["abi3-py311"] }
+wlr-libpy = { git = "https://github.com/vmware-labs/webassembly-language-runtimes.git", branch="rust-py-example", features = ["py_main"] }
```

Comment:
```suggestion
wlr-libpy = { path = "../../../wlr-libpy", features = ["py_main"] }
```

Reply:
You are correct. This is what I used during development. However, my intention is to have this crate consumable by external parties. Thus I wanted to make the example in a way that allows people to copy-paste toml lines and get to use the crate straightaway.

Reply:
I just confirmed that Cargo fetches the [entire repository and look for any packages inside](https://github.com/rust-lang/cargo/issues/1462#issuecomment-87002475). @assambar, the `path` approach should work on both local and remote as `cargo` will pull the entire repo.

This is the same approach we're using in [Wasm Workers Server](https://github.com/vmware-labs/wasm-workers-server/blob/main/Cargo.toml#L76).

Reply:
Ok, so I will use the path from wlr-libpy to wlr-assets, but plan on keeping the git repo approach for the example itself.